dapp_runner/runner/runner.py

```diff
@@ -204,10 +204,10 @@ def dapp_state(self) -> dict:
 
         return {
             cluster_id: {
-                idx: self.clusters[cluster_id].instances[idx].state.name
-                for idx in range(len(self.clusters[cluster_id].instances))
+                instance_number: instance.state.name
```

Comment:
I'd leave "index" here -> just rename it to `instance_index` if you feel `idx` is too terse

explanation: just that I think the word "index" has a very clear meaning and it's obvious why it starts with `0` and not `1` ... a number could have a less exact meaning - specifically, it could be thought of as the number _of_ instances ... which would be entirely misleading...